src/PanelTraits/Boxes.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Backpack\CRUD\PanelTraits;
+
+trait Boxes
+{
+    public $sideBoxesEnabled = false;
+    public $boxesOptions = [];
+
+    public function enableSideBoxes()
+    {
+        $this->sideBoxesEnabled = true;
+
+        return $this->sideBoxesEnabled;
+    }
+
+    public function disableSideBoxes()
+    {
+        $this->sideBoxesEnabled = false;
+
+        return $this->sideBoxesEnabled;
+    }
+
+    public function sideBoxesEnabled()
+    {
+        return $this->sideBoxesEnabled;
+    }
+
+    public function sideBoxesDisabled()
```

Comment:
This method name is also inspired from Tabs trait. I think you have to be consistent.
https://github.com/Laravel-Backpack/CRUD/blob/master/src/PanelTraits/Tabs.php#L30